docs/DeveloperGuide.md

```diff
@@ -398,20 +395,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1.  Should work on any _mainstream OS_ as long as it has Java `17` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-4.  Codebase should be modular and easy to update, so that future developers can easily add new features or fix bugs.
+4.  Codebase should be _modular_ and easy to update, so that future developers can easily add new features or fix bugs.
 5.  App should have the ability to gracefully handle a data file that is corrupted without crashing, and inform the user of the issue.
-6.  System should gracefully recover from minor errors (e.g., incorrect or incomplete input) by either prompting the user to correct the issue, or using default values where applicable.
+6.  System should _gracefully recover_ from minor errors (e.g., incorrect or incomplete input) by either prompting the user to correct the issue, or using default values where applicable.
 7.  System should have well-written and accessible user and developer documentation to support both end-users and future developers in using and maintaining the application.
 8.  System should have a comprehensive suite of automated tests to ensure that new changes do not break existing functionality.
-9.  System should implement a data retention policy, ensuring that data and records older than a certain period are archived or deleted to optimize performance and comply with legal requirements.
+9.  System should implement a _data retention policy_, ensuring that data and records older than a certain period are archived or deleted to optimize performance and comply with legal requirements.
 10. System should have easy-to-use backup and restore functionality, enabling users to create backups of their data, and restore their data in the event of data loss or corruption.
 
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Modular**: Codebase is divided into separate parts that can be updated without affecting other parts.
+* **Gracefully recover**: Provides an appropriate solution without crashing the application.
 * **Data retention policy**: Guidelines regarding what data should be kept, how long data should be kept and more.
-* **CLI application**: Any application that takes in input via text, not necessarily through a command console.
+* **CLI - interface**: Any application that mainly takes in input via text, not necessarily through a command console.
```

Comment:
Perhaps you could emphasize that the interaction with the application is via text?

Reply:
Ok! Fixed.